force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -144,6 +149,20 @@ public inherited sharing class RD2_RecurringDonation {
         return this;
     }
 
+    /**
+    * @description Stores the Change Type in the object, then sets it to null on the Recurring Donation
+    * @return RD2_RecurringDonation This record instance
+    */
+    public RD2_RecurringDonation storeAndClearChangeType() {
```

Comment:
My reason for doing this is to have the Change Type cleared before the actual Recurring Donation update, but still have the value itself available in the logic for Change History records.